Dockerfile

```diff
@@ -1,7 +1,8 @@
-ARG GO_VERSION=1.21-bullseye
+ARG GO_VERSION=1.21-bookworm
```

Comment:
for sycl I might need to switch to ubuntu 22.04 LTS - would that make things easier for ROCm too?